tests/test-cases/Group1-Docker-Commands/1-43-Docker-CP.md

```diff
@@ -0,0 +1,80 @@
+Test 1-43 - Docker CP
```

Comment:
This whole effort got a bit big, I would actually prefer to break this up into 2 cases - 1-43 - Docker CP Offline, 1-44 - Docker CP Online.  That would make this much easier to read and more manageable and I think it is a good, natural break.